sympy/solvers/tests/test_solveset.py

```diff
@@ -53,15 +54,19 @@ def test_invert_real():
     x = Symbol('x', real=True)
     y = Symbol('y')
     n = Symbol('n')
+
     def ireal(x, s=S.Reals):
         return Intersection(s, x)
 
     # issue 14223
     assert invert_real(x, 0, x, Interval(1, 2)) == (x, S.EmptySet)
 
-    minus_n = Intersection(Interval(-oo, 0), FiniteSet(-n))
-    plus_n = Intersection(Interval(0, oo), FiniteSet(n))
-    assert solveset(abs(x) - n, x, S.Reals) == Union(minus_n, plus_n)
+    assert solveset(abs(x) - n, x, S.Reals) == ConditionSet(
+        x, Contains(n, Interval(0, oo), evaluate=False), {-n, n})
+
+    conditions = Contains(n, Interval(0, oo), evaluate=False)
+    assert solveset(abs(x) - n, x, S.Reals) == ConditionSet(
```

Comment:
tests are repeated.

Reply:
Also `evaluate=False` should be removed now (as per the changes done in `Contains` class above).

Reply:
Will correct when I update PR.